### ISSUE TITLE
Deal with NEON records that have multiple sequencing runs

### ISSUE THREAD

**Issue by aclum** [microbiomedata/issues#572]
**Deliverable this task is associated with**

_See Deliverables tab here: 

- 1,2,3

**RACI**

_Tag people in their roles_
- Responsible: 
- Accountable:
- Consulted:
- Informed:

**Describe the the task**
- [ ] [What are the tasks?]

**Criteria for completion**
- [What does success look like for this task to be considered complete?]

**Estimate people time**
- [Hours or days of people time. 1 person, 4 hours]

**Completion Date (Goal)**
- [GOAL!]

**Target Sprint Start & End Dates**
- Start:
- End:

**Tag Blocker/Contingent upon issues**
- blocked on Berkeley schema which allows for parthood to connect multiple OmicsProcesing/DataGeneration records.

**Comment by aclum** [comment]
Still blocked, this was rolled back from the berkeley schema, Katherine has a [PR](https://github.com/microbiomedata/nmdc-schema/pull/2230) which adds manifest set but ELT and workflow code needs to be aware of this.

**Comment by aclum** [comment]
check if this is a duplicate of https://github.com/microbiomedata/issues/issues/1672

**Comment by aclum** [comment]
resolved by 
- [ ] https://github.com/microbiomedata/issues/issues/1515
- [ ] https://github.com/microbiomedata/issues/issues/1390
